diff -crB old/.harmattan-themes/Numix/God-Mode/normal-mode/.conkyrc new/.harmattan-themes/Numix/God-Mode/normal-mode/.conkyrc
*** old/.harmattan-themes/Numix/God-Mode/normal-mode/.conkyrc	2017-05-19 10:17:27.000000000 -0400
--- new/.harmattan-themes/Numix/God-Mode/normal-mode/.conkyrc	2017-06-01 20:56:52.991262586 -0400
***************
*** 22,38 ****
  -------------------------------------
  --  Window Specifications
  -------------------------------------
! gap_x=100,
! gap_y=70,
  minimum_height=620,
  minimum_width=268,
  own_window=true,
  own_window_type="normal",
  own_window_transparent=true,
  own_window_hints="undecorated,below,sticky,skip_taskbar,skip_pager",
  border_inner_margin=0,
  border_outer_margin=0,
! --alignment="middle_middle",
  --own_window_argb_visual=true,
  --own_window_argb_value=0,
  
--- 22,39 ----
  -------------------------------------
  --  Window Specifications
  -------------------------------------
! gap_x=0,
! gap_y=-10,
  minimum_height=620,
  minimum_width=268,
  own_window=true,
  own_window_type="normal",
  own_window_transparent=true,
  own_window_hints="undecorated,below,sticky,skip_taskbar,skip_pager",
+ own_window_class="conky",
  border_inner_margin=0,
  border_outer_margin=0,
! alignment="top_right",
  --own_window_argb_visual=true,
  --own_window_argb_value=0,
  
***************
*** 60,69 ****
  color2="DCDCDC",
  color3="DCDCDC",
  color4="F9F9F9",
! color5="D64937",
  color6="888888",
  color7="484848",
! color8="2D2D2D",
  
  
  -------------------------------------
--- 61,70 ----
  color2="DCDCDC",
  color3="DCDCDC",
  color4="F9F9F9",
! color5="1685A6",
  color6="888888",
  color7="484848",
! color8="1B2B34",
  
  
  -------------------------------------
***************
*** 88,94 ****
  --  Locale (e.g. "es_ES.UTF-8")
  --  Leave empty for default
  -------------------------------------
! template9=""
  
  }
  
--- 89,95 ----
  --  Locale (e.g. "es_ES.UTF-8")
  --  Leave empty for default
  -------------------------------------
! template9="en_US.UTF-8"
  
  }
  
***************
*** 159,165 ****
  ${voffset 4}${alignr 39}${color6}${uptime_short}${color}
  ${voffset -47}${alignc}${color2}${cpubar 5,36}${color}
  ${voffset 4}${alignc}${color2}${membar 5,36}${color}
! ${voffset 29}${goto 40}${loadgraph 26,190 D64937 D64937 -l}
  \
  \
  \
--- 160,166 ----
  ${voffset 4}${alignr 39}${color6}${uptime_short}${color}
  ${voffset -47}${alignc}${color2}${cpubar 5,36}${color}
  ${voffset 4}${alignc}${color2}${membar 5,36}${color}
! ${voffset 29}${goto 40}${loadgraph 26,190 1685A6 1685A6 -l}
  \
  \
  \
***************
*** 188,240 ****
  \
  ${if_existing /proc/net/route ppp0}
  ${voffset -227}${goto 40}${color5}Up: ${color2}${upspeed ppp0}${color5}${goto 150}Down: ${color2}${downspeed ppp0}
! ${voffset 10}${goto 40}${upspeedgraph ppp0 26,80 d64937 d64937}${goto 150}${downspeedgraph ppp0 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup ppp0}${color5}${goto 150}Received: ${color2}${totaldown ppp0}
  ${else}
  ${if_existing /proc/net/route ppp1}
  ${voffset -240}${goto 40}${color5}Up: ${color2}${upspeed ppp1}${color5}${goto 150}Down: ${color2}${downspeed ppp1}
! ${voffset 10}${goto 40}${upspeedgraph ppp1 26,80 d64937 d64937}${goto 150}${downspeedgraph ppp1 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup ppp1}${color5}${goto 150}Received: ${color2}${totaldown ppp1}
  ${else}
  ${if_existing /proc/net/route wlp2s1}
  ${voffset -253}${goto 40}${color5}Up: ${color2}${upspeed wlp2s1}${color5}${goto 150}Down: ${color2}${downspeed wlp2s1}
! ${voffset 10}${goto 40}${upspeedgraph wlp2s1 26,80 d64937 d64937}${goto 150}${downspeedgraph wlp2s1 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlp2s1}${color5}${goto 150}Received: ${color2}${totaldown wlp2s1}
  ${else}
  ${if_existing /proc/net/route wlp2s0}
  ${voffset -266}${goto 40}${color5}Up: ${color2}${upspeed wlp2s0}${color5}${goto 150}Down: ${color2}${downspeed wlp2s0}
! ${voffset 10}${goto 40}${upspeedgraph wlp2s0 26,80 d64937 d64937}${goto 150}${downspeedgraph wlp2s0 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlp2s0}${color5}${goto 150}Received: ${color2}${totaldown wlp2s0}
  ${else}
  ${if_existing /proc/net/route wlan0}
  ${voffset -279}${goto 40}${color5}Up: ${color2}${upspeed wlan0}${color5}${goto 150}Down: ${color2}${downspeed wlan0}
! ${voffset 8}${goto 40}${upspeedgraph wlan0 26,80 d64937 d64937}${goto 150}${downspeedgraph wlan0 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlan0}${color5}${goto 150}Received: ${color2}${totaldown wlan0}
  ${else}
  ${if_existing /proc/net/route wlan1}
  ${voffset -292}${goto 40}${color5}Up: ${color2}${upspeed wlan1}${color5}${goto 150}Down: ${color2}${downspeed wlan1}
! ${voffset 10}${goto 40}${upspeedgraph wlan1 26,80 d64937 d64937}${goto 150}${downspeedgraph wlan1 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlan1}${color5}${goto 150}Received: ${color2}${totaldown wlan1}
  ${else}
  ${if_existing /proc/net/route eth1}
  ${voffset -305}${goto 40}${color5}Up: ${color2}${upspeed eth1}${color5}${goto 150}Down: ${color2}${downspeed eth1}
! ${voffset 10}${goto 40}${upspeedgraph eth1 26,80 d64937 d64937}${goto 150}${downspeedgraph eth1 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup eth1}${color5}${goto 150}Received: ${color2}${totaldown eth1}
  ${else}
  ${if_existing /proc/net/route eth0}
  ${voffset -318}${goto 40}${color5}Up: ${color2}${upspeed eth0}${color5}${goto 150}Down: ${color2}${downspeed eth0}
! ${voffset 10}${goto 40}${upspeedgraph eth0 26,80 d64937 d64937}${goto 150}${downspeedgraph eth0 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup eth0}${color5}${goto 150}Received: ${color2}${totaldown eth0}
  ${else}
  ${if_existing /proc/net/route enp0s0}
  ${voffset -331}${goto 40}${color5}Up: ${color2}${upspeed enp0s0}${color5}${goto 150}Down: ${color2}${downspeed enp0s0}
! ${voffset 10}${goto 40}${upspeedgraph enp0s0 26,80 d64937 d64937}${goto 150}${downspeedgraph enp0s0 26,80 d64937 d64937}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup enp0s0}${color5}${goto 150}Received: ${color2}${totaldown enp0s0}
  ${else}
! ${if_existing /proc/net/route enp0s1}
! ${voffset -344}${goto 40}${color5}Up: ${color2}${upspeed enp0s1}${color5}${goto 150}Down: ${color2}${downspeed enp0s1}
! ${voffset 10}${goto 40}${upspeedgraph enp0s1 26,80 d64937 d64937}${goto 150}${downspeedgraph enp0s1 26,80 d64937 d64937}
! ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup enp0s1}${color5}${goto 150}Received: ${color2}${totaldown enp0s1}
  ${else}
  ${voffset -311}${goto 40}${color5}Network disconnected${color}
  ${image ~/.harmattan-assets/misc/Numix/God-Mode/offline.png -p 44,284 -s 16x16}
--- 189,241 ----
  \
  ${if_existing /proc/net/route ppp0}
  ${voffset -227}${goto 40}${color5}Up: ${color2}${upspeed ppp0}${color5}${goto 150}Down: ${color2}${downspeed ppp0}
! ${voffset 10}${goto 40}${upspeedgraph ppp0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph ppp0 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup ppp0}${color5}${goto 150}Received: ${color2}${totaldown ppp0}
  ${else}
  ${if_existing /proc/net/route ppp1}
  ${voffset -240}${goto 40}${color5}Up: ${color2}${upspeed ppp1}${color5}${goto 150}Down: ${color2}${downspeed ppp1}
! ${voffset 10}${goto 40}${upspeedgraph ppp1 26,80 1685A6 1685A6}${goto 150}${downspeedgraph ppp1 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup ppp1}${color5}${goto 150}Received: ${color2}${totaldown ppp1}
  ${else}
  ${if_existing /proc/net/route wlp2s1}
  ${voffset -253}${goto 40}${color5}Up: ${color2}${upspeed wlp2s1}${color5}${goto 150}Down: ${color2}${downspeed wlp2s1}
! ${voffset 10}${goto 40}${upspeedgraph wlp2s1 26,80 1685A6 1685A6}${goto 150}${downspeedgraph wlp2s1 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlp2s1}${color5}${goto 150}Received: ${color2}${totaldown wlp2s1}
  ${else}
  ${if_existing /proc/net/route wlp2s0}
  ${voffset -266}${goto 40}${color5}Up: ${color2}${upspeed wlp2s0}${color5}${goto 150}Down: ${color2}${downspeed wlp2s0}
! ${voffset 10}${goto 40}${upspeedgraph wlp2s0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph wlp2s0 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlp2s0}${color5}${goto 150}Received: ${color2}${totaldown wlp2s0}
  ${else}
  ${if_existing /proc/net/route wlan0}
  ${voffset -279}${goto 40}${color5}Up: ${color2}${upspeed wlan0}${color5}${goto 150}Down: ${color2}${downspeed wlan0}
! ${voffset 8}${goto 40}${upspeedgraph wlan0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph wlan0 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlan0}${color5}${goto 150}Received: ${color2}${totaldown wlan0}
  ${else}
  ${if_existing /proc/net/route wlan1}
  ${voffset -292}${goto 40}${color5}Up: ${color2}${upspeed wlan1}${color5}${goto 150}Down: ${color2}${downspeed wlan1}
! ${voffset 10}${goto 40}${upspeedgraph wlan1 26,80 1685A6 1685A6}${goto 150}${downspeedgraph wlan1 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup wlan1}${color5}${goto 150}Received: ${color2}${totaldown wlan1}
  ${else}
  ${if_existing /proc/net/route eth1}
  ${voffset -305}${goto 40}${color5}Up: ${color2}${upspeed eth1}${color5}${goto 150}Down: ${color2}${downspeed eth1}
! ${voffset 10}${goto 40}${upspeedgraph eth1 26,80 1685A6 1685A6}${goto 150}${downspeedgraph eth1 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup eth1}${color5}${goto 150}Received: ${color2}${totaldown eth1}
  ${else}
  ${if_existing /proc/net/route eth0}
  ${voffset -318}${goto 40}${color5}Up: ${color2}${upspeed eth0}${color5}${goto 150}Down: ${color2}${downspeed eth0}
! ${voffset 10}${goto 40}${upspeedgraph eth0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph eth0 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup eth0}${color5}${goto 150}Received: ${color2}${totaldown eth0}
  ${else}
  ${if_existing /proc/net/route enp0s0}
  ${voffset -331}${goto 40}${color5}Up: ${color2}${upspeed enp0s0}${color5}${goto 150}Down: ${color2}${downspeed enp0s0}
! ${voffset 10}${goto 40}${upspeedgraph enp0s0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph enp0s0 26,80 1685A6 1685A6}
  ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup enp0s0}${color5}${goto 150}Received: ${color2}${totaldown enp0s0}
  ${else}
! ${if_existing /proc/net/route enp5s0}
! ${voffset -344}${goto 40}${color5}Up: ${color2}${upspeed enp5s0}${color5}${goto 150}Down: ${color2}${downspeed enp5s0}
! ${voffset 10}${goto 40}${upspeedgraph enp5s0 26,80 1685A6 1685A6}${goto 150}${downspeedgraph enp5s0 26,80 1685A6 1685A6}
! ${voffset 9}${goto 40}${color5}Sent: ${color2}${totalup enp5s0}${color5}${goto 150}Received: ${color2}${totaldown enp5s0}
  ${else}
  ${voffset -311}${goto 40}${color5}Network disconnected${color}
  ${image ~/.harmattan-assets/misc/Numix/God-Mode/offline.png -p 44,284 -s 16x16}
